docs/tutorial/Basic/tensor.ipynb

```diff
@@ -0,0 +1,1279 @@
+{
```

Comment:
<p>you can put the table in a <a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting " target="_blank">markdown code block</a>) if it is not yet in one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>